packages/react/README.md

```diff
@@ -11,9 +11,22 @@ See full documentation [here](https://docs.novu.co/inbox/react/get-started).
 npm install @novu/react
 ```
 
-## Getting Started
+## Try it instantly (Keyless mode)
 
-- Add the below code in the app.tsx file
+The keyless mode is designed for local testing and experimentation, letting you use Novu's Inbox component without any configuration. Just import and render.
+
+```jsx
+import React from 'react';
+import { Inbox } from '@novu/react';
+
+export function App() {
+  return <Inbox />;
+}
+```
+
+## Connect to real subscribers 
+
+To connect the Inbox component with your Novu environment and real subscribers, set the `applicationIdentifier` and `subscriberId`
```

Comment:
@Aviatorscode2 could you also update the lower snippet to use the subscriber and app id props without the options object to align with our other snippets?